pyanaconda/bootloader.py

```diff
@@ -1010,6 +1009,9 @@ class GRUB(BootLoader):
 
     _serial_consoles = ["ttyS"]
 
+    def _config_dir(self):
```

Comment:
@jaymzh: where did this come from? I don't remember this being in the original patch and I don't see it in master. It causes anaconda to die with a can't concatenate a string with an object (or something)